src/components/Icon/iconInlineTop.ios.js

```diff
@@ -0,0 +1 @@
+export default 1;
```

Comment:
This is specially used for Styling purposes and the file name does not suit the purpose. we prefer Styles in filename for such scenarios.
A good file name would be `iconWrapperStyles.js`. 
Then convert this into 
```suggestion
export default {top: 1};
```
So that it can be used as iconWrapperStyles in the Icon component.

Reply:
@parasharrajat good idea!